boto3/setup.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+"""
+distutils/setuptools install script.
+"""
+import os
+import re
+
+from setuptools import setup, find_packages
+
+
+ROOT = os.path.dirname(__file__)
+VERSION_RE = re.compile(r'''__version__ = ['"]([0-9.]+)['"]''')
+
+
+requires = [
+    'botocore>=1.20.105,<1.21.0',
+    'jmespath>=0.7.1,<1.0.0',
+    's3transfer>=0.4.0,<0.5.0'
+]
+
+
+def get_version():
+    init = open(os.path.join(ROOT, 'boto3', '__init__.py')).read()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the resource returned by the following method call: *open*. The resource is allocated by call *builtins.open*. Execution paths that do not contain closure statements were detected. To prevent this resource leak, close the object returned by *open()* in a try-finally block or declare it using a `with` statement.

**More info**
[View details about the `with` statement in the Python developer&#39;s guide](https://www.python.org/dev/peps/pep-0343/) (external link).